collectd_openstack/gnocchi/sender.py

```diff
@@ -124,50 +133,61 @@ def _get_metric_id(self, metername, unit):
             if metric_id is not None:
                 self._meter_ids[metername] = metric_id
             else:
-                self._meter_ids[metername] = self._create_metric(metername, endpoint, unit)
+                self._meter_ids[metername] = self._create_metric(
+                    metername, endpoint, unit)
 
         return self._meter_ids[metername]
 
     def _get_metric(self, metername, endpoint, unit):
         if self.resource_id is None:
             return None
-        url = "{}/v1/resource/generic/{}/metric/".format(endpoint, self.resource_id)
+
+        url = "{}/v1/resource/{}/{}/metric/".format(
+            endpoint, self.meter_type, self.resource_id)
         try:
-            result = self._perform_request(url, None, self._auth_token, req_type="get")
+            result = self._perform_request(
+                url, None, self._auth_token, req_type="get")
             metrics = json.loads(result.text)
-            metrics = [m for m in metrics if m['name'] == metername and m['unit'] == unit]
+            metrics = [
+                m for m in metrics
+                if m['name'] == metername and m['unit'] == unit]
             metric_id = metrics[0]['id']
-        except:
+        except Exception:
```

Comment:
Just wondering, is this PEP style? I usually just write `except:`

Reply:
Yes. I blindly follow my linter on atom.